src/main/resources/application.yml

```diff
@@ -6,6 +6,8 @@ hostiflix-login-redirect:
 
 deploymentServiceUrl: http://deployment-service/jobs/submit
 
+webhookPayloadUrl: https://api.hostiflix.com/github/webhook
```

Comment:
What about a more meaningful name like `hostiflix-github-webhook`? Like `hostiflix-login-redirect`. To show that this endpoint is on our side?